src/client.rs

```diff
@@ -107,70 +137,82 @@ impl Client {
     /// server remembers the state of the subscription. This allows for many
     /// different modes of operations compared to a regular or catchup
     /// subscription where the client holds the subscription state.
-    pub fn create_persistent_subscription<S>(
+    pub async fn create_persistent_subscription<StreamName, GroupName>(
         &self,
-        stream_id: S,
-        group_name: S,
-    ) -> commands::CreatePersistentSubscription
+        stream_id: StreamName,
```

Comment:
**SUGGESTION:** can we align the function property naming? Sometimes it uses `stream`, sometimes `stream_id`. I'd suggest using `stream_name` as it is in other clients.

Reply:
Done